src/Compilers/Test/Core/FX/MonoHelpers.cs

```diff
@@ -11,5 +11,6 @@ namespace Microsoft.CodeAnalysis.Test.Utilities
     public static class MonoHelpers
     {
         public static bool IsRunningOnMono() => Roslyn.Test.Utilities.ExecutionConditionUtil.IsMonoDesktop;
+        public static bool IsUsingMonoRuntime() => Roslyn.Test.Utilities.ExecutionConditionUtil.IsMonoRuntime;
```

Comment:
Please make the naming a more crisp here to denote if it's any, desktop or core mono.

Reply:
How about using "IsRUnningOnMonoCore" ? 
Let me know if these names looks good to you ? I would like to commit my next change once we are in agreement with names to be used .